deploy/crd/bases/vault.banzaicloud.com_vaults.yaml

```diff
@@ -1150,7 +1150,6 @@ spec:
                     required:
                     - keyLabel
                     - modulePath
-                    - pin
```

Comment:
@mark64 was this changed manually? If so, you can add `// +optional` above https://github.com/mark64/vault-operator/blob/cacd2d5abdbd51cef162683e4e9b4e5e051348ed/pkg/apis/vault/v1alpha1/vault_types.go#L916 as we use https://book.kubebuilder.io/reference/controller-gen to generate CRD from specs.

Reply:
Thanks, I ran `make gen-manifest` to update things, it added a restart policy though. Should that have happened?

Reply:
That's okay, the new version added it automatically. It's still not a required field so nothing should change.